ui/hooks/src/use-posts.new.ts

```diff
@@ -203,8 +203,8 @@ export function useDeletePost(postID: string) {
       });
       return { previousPost };
     },
-    onSuccess: () => {
-      queryClient.invalidateQueries([ENTRY_KEY, postID]);
+    onSuccess: async () => {
```

Comment:
why make it async? I see in the docs it's used sync https://react-query.tanstack.com/guides/invalidations-from-mutations

Reply:
the docs might be outdated, check the signature from https://github.com/tannerlinsley/react-query/blob/a53f6325e9761b1435b85cfbd337ea1a75584dde/src/core/queryClient.ts#L241

Reply:
`queryClient.invalidateQueries('key')` returns a `Promise`